--- resolv_wrapper-1.0.0/ConfigureChecks.cmake.orig	2014-10-24 14:52:05.000000000 +0200
+++ resolv_wrapper-1.0.0/ConfigureChecks.cmake	2014-11-01 15:50:06.263825347 +0100
@@ -50,6 +50,8 @@
 check_include_file(sys/types.h HAVE_SYS_TYPES_H)
 check_include_file(resolv.h HAVE_RESOLV_H)
 
+set(CMAKE_REQUIRED_LIBRARIES)
+
 # FUNCTIONS
 find_library(RESOLV_LIRBRARY resolv)
 
@@ -92,8 +94,6 @@
 check_function_exists(res_nsearch HAVE_RES_NSEARCH)
 check_function_exists(__res_nsearch HAVE___RES_NSEARCH)
 
-set(CMAKE_REQUIRED_LIBRARIES)
-
 if (UNIX)
     if (NOT LINUX)
         # libsocket (Solaris)
